packages/gatsby-theme/src/components/Share/ShareContainer.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types"
+import { Box } from "theme-ui"
+
+const ShareContainer = ({ children, ...props }) => {
+  return (
+    <Box
+      __css={{
+        display: "flex",
+        alignItems: "center",
+        color: "darkShade",
+        button: {
```

Comment:
This styles are affecting the button inside the container, can we move this to the `ShareSocial.js` itself instead? I think it will make more sense